src/common/HostInterface.c

```diff
@@ -430,22 +429,15 @@ void ProcessCommandFromHost(char * strCMD)
 
 		if (ptrParams == 0)
 		{
-			sprintf(cmdReply, "%s %d", strCMD, ConsoleLogLevel);
+			snprintf(cmdReply, sizeof(cmdReply), "%s %d", strCMD, ardop_log_get_level_console());
 			SendReplyToHost(cmdReply);
 		}
 		else
 		{
-			i = atoi(ptrParams);
```

Comment:
If ptrParams is not a number, atoi(ptrParams) returns 0 which ends up setting ConsoleLogLevel to ZF_LOG_VERBOSE which is not desirable in most cases.  Consider using strtol() or some other alternative approach that detects a non-numerical value, in which case a Syntax Error should be sent  to the Host program.  Also for LOGLEVEL